klaviyo/lists.py

```diff
@@ -225,3 +226,23 @@ def get_all_members(self, group_id, marker=None):
         params = self._build_marker_param(marker)
 
         return self._v2_request('group/{}/{}/{}'.format(group_id, self.MEMBERS, self.ALL), self.HTTP_GET, params)
+
+    def get_members_from_segment(self, segment_id, emails):
+        """
+        Checks if one or more emails are in a given segment.
+        No distinction is made between a person not being in a given segment,
+        and not being present in Klaviyo at all.
+        Can check up to a maximum of 100 emails at a time.
+
+        https://apidocs.klaviyo.com/reference/lists-segments#get-segment-members
+
+        segment_id (str): The segment id.
+        emails (list):  A list of email addresses.
+
+        """
+
+        params = {
+            self.EMAIL: emails
```

Comment:
The list of emails needs to be converted to a comma-delimited string here otherwise the requests library will convert the params into multiple 'email' query params and the Klaviyo api will only return one of the values.

Reply:
done.